content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+---
+title: GreenLake Platform Infrastructure monitoring with Apache SkyWalking
+date: 2022-12-13T08:52:15.545Z
+author: Thirukkannan M
+authorimage: /img/thirupp_192x192.jpg
+thumbnailimage: /img/thirupp_192x192.jpg
+disable: false
+tags:
+  - Infrastructure Monitoring
+  - Apache SkyWalking
+---
+**Overview**
```

Comment:
Hi there. If you've noticed, we have a lot of blog posts recently that have come in that start off with exactly the same Overview as what you have here. Since this is a blog, and not product documentation, we need to work on attracting the reader to your post. Telling them what it is you're about to cover in your blog and how it will help them/benefit them is how you will gain their attention. 

I took a look at your conclusion to try and figure out what this blog post was really about. You need to bring this type of information up to the top. You can integrate some of the other material you have in your overview into it, but you really need to tell the reader WHY they want to read your blog post and not someone else's. (well, you can help them want to read someone else's, too by virtue of mentioning them in the conclusion, but only after they've read yours :-)). I'll see what I can do below to help you understand what I mean.